bokeh/validation/check.py

```diff
@@ -0,0 +1,30 @@
+''' Provide the ``check_integrity`` function.
+
+'''
+from __future__ import absolute_import
+
+import logging
+logger = logging.getLogger(__file__)
+
+from ..settings import settings
+from .exceptions import ValidationError
+
+def check_integrity(models):
+
+    messages = dict(error=[], warning=[])
+
+    for model in models:
+        validators = [
+            obj for obj in model.__class__.__dict__.values() if getattr(obj, "validator_type", None)
+        ]
```

Comment:
Why not `dir()` (on the object or class)? Otherwise, I think I would walk the MRO. Any higher-level solution is probably walking the mro anyway.